docs/installation.rst

```diff
@@ -40,10 +40,224 @@ With that, you should be ready to use ``openfe``!
 Single file installer
 ~~~~~~~~~~~~~~~~~~~~~
 
-[COMING SOON!]
-
-.. TODO: maybe Mike can fill this in? just needs (1) how to download the
-   single file installer; (2) how to use the single file installer
+Single file installers are available for x86_64 Linux and MacOS. 
```

Comment:
This is a lot of lines for this section - any chance we might be willing to move it to a different file that we then just link to here?